src/cim/types/k8s/cim-infra-env.ts

```diff
@@ -1,6 +1,6 @@
 import { K8sResourceCommon, Selector } from 'console-sdk-ai-lib';
 
-export type InfraEnv = K8sResourceCommon & {
+export type CIMInfraEnv = K8sResourceCommon & {
```

Comment:
@jtomasek, @mareklibra , @rawagner Please review this change ^

Reply:
A better name would be
```suggestion
export type InfraEnvK8sResource = K8sResourceCommon & {
```

which aligns with name of other k8s resources